### PR TITLE
*: Support libraries in cli runner

### LINKED ISSUE #17: Support packages (libraries) in R
Renjin [supports some of CRAN and BioConductor R libraries](http://packages.renjin.org/), Processing.R could benefit from it.

ref [the post in forum](https://forum.processing.org/two/discussion/comment/91251/#Comment_91251)

> As for libraries, I did a feasibility study. Renjin is used in Processing.R as the backend to interpret R code in JVM. [Documentation in renjin.org](http://docs.renjin.org/en/latest/interactive/index.html#using-packages) shows differences about packages between GNU R and renjin. 
>
>It [supports some of CRAN and BioConductor R libraries](http://packages.renjin.org/) and I want to integrate those libraries into Processing.R in GSoC. But I'm not sure whether those packages could be installed in runtime or ahead of time.
>
>BTW I think Jython supports those libraries which do not require C compilation, although I do not have the experience on it.

### PR DESCRIPTION
**Pull request by gaocegege**
Close #17 

Signed-off-by: Ce Gao <ce.gao@outlook.com>

### CODE REVIEW

**Comment by jeremydouglass** [comment]
I tested this and it worked with the default PeasyCam example:

```
java -jar ./try/RLangMode.jar /Users/myusername/git/Processing.R/examples/Libraries/LibraryBasics/LibraryBasics.rpde
```

Currently the only instructions on how to use the CLI are in compilation.md.

https://github.com/gaocegege/Processing.R/blob/master/raw-docs/devel/compilation.md

Is the Runner included with the release, and if so should use-instructions be included in the top-level documentation?

**Comment by gaocegege** [comment]
Yeah, it is a good idea. But IMO the runner is not a common way for users. It is just an early way to have a test on Processing.R.

File: src/rprocessing/lancher/StandaloneSketch.java [hunk 1]
```diff
@@ -103,44 +105,26 @@ public StandaloneSketch(final String[] args) throws Exception {
...
+  private File getLibraryDir() {
+    if (Platform.isMacOS()) {
+      String home = System.getProperty("user.home");
+      return new File(home + "/Documents/Processing/libraries");
+    } else if (Platform.isLinux()) {
+      String home = System.getProperty("user.home");
+      return new File(home + "/sketchbook/libraries");
+    } else if (Platform.isWindows()) {
+      String home = System.getProperty("user.home");
+      return new File(home + "\\Documents\\Processing\\modes");
```
Comment (jeremydouglass) [inline src/rprocessing/lancher/StandaloneSketch.java]: modes? Shouldn't this be libraries?

Comment (gaocegege) [inline src/rprocessing/lancher/StandaloneSketch.java]: It is a typo. Thanks for your review :thinking:

**Pull request merged**